app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -524,6 +522,31 @@ def traces_helpers(search:)
         ::Katello::HostTracer.helpers_for(traces)
       end
 
+      def probably_rhel?
+        # Get the os name from sub-man facts rather than operatingsystem. This is
+        # less likely to have been changed by the user.
+        os_name, = facts('distribution::name').values # only query for that one fact, then get its value
```

Comment:
I'm using `facts('distribution::name')` here instead of `facts['distribution::name']` because this guarantees that the SQL query will only retrieve that one fact, not all facts for the host.

With the comma, this line is equivalent to 

```
os_name = facts('distribution::name').values.first
```

but avoids the need for safe navigation (`&.`).